src/Pages/Shared/index.ts

```diff
@@ -0,0 +1,3 @@
+import LinkedCIDetail from './LinkedCIDetailsModal/LinkedCIDetail'
+
+export default LinkedCIDetail
```

Comment:
Move to LinkedCIDetailsModal

```suggestion
export { default } from './LinkedCIDetail'
```